controllers/glanceapi_controller.go

```diff
@@ -445,6 +447,41 @@ func (r *GlanceAPIReconciler) reconcileNormal(ctx context.Context, instance *gla
 	// TODO check when/if Init, Update, or Upgrade should/could be skipped
 	//
 
+	// Get Enabled backends from customServiceConfig and run pre backend conditions
+	availableBackends := glance.GetEnabledBackends(instance.Spec.CustomServiceConfig)
+	// iterate over availableBackends for backend specific cases
+	for i := 0; i < len(availableBackends); i++ {
+		backendToken := strings.SplitN(availableBackends[i], ":", 2)
+		switch {
+		case backendToken[1] == "cinder":
+			// get admin
+			/* var err error
```

Comment:
@konan-abhi this part I commented isn't properly working and I get:

```
2023-07-05T07:55:17-04:00       INFO    Shutdown signal received, waiting for all workers to finish     {"controller": "glanceapi", "controllerGroup": "glance.openstack.org", "controllerKind": "GlanceAPI"}
2023-07-05T07:55:17-04:00       INFO    Shutdown signal received, waiting for all workers to finish     {"controller": "glance", "controllerGroup": "glance.openstack.org", "controllerKind": "Glance"}
2023-07-05T07:55:17-04:00       INFO    All workers finished    {"controller": "glance", "controllerGroup": "glance.openstack.org", "controllerKind": "Glance"}
2023-07-05T07:55:17-04:00       INFO    controllers.GlanceAPI   Checking volume service is running or not
2023-07-05T07:55:17-04:00       INFO    controllers.GlanceAPI   SERVICES: <nil>
2023-07-05T07:55:17-04:00       ERROR   Reconciler error        {"controller": "glanceapi", "controllerGroup": "glance.openstack.org", "controllerKind": "GlanceAPI", "GlanceAPI": {"name":"glance-external","namespace":"openstack"}, "namespace": "openstack", "name": "glance-external", "reconcileID": "0824908b-ae1a-4b3b-b9f9-c155a1b398b9", "error": "Resource not found: [GET http://keystone-public-openstack.apps-crc.testing/v3/os-services], error message: <!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 3.2 Final//EN\">\n<title>404 Not Found</title>\n<h1>Not Found</h1>\n<p>The requested URL was not found on the server. If you entered
the URL manually please check your spelling and try again.</p>\n"}
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler
        /home/stack/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.6/pkg/internal/controller/controller.go:329
```
coming from lib-common [1].
While we're working on those checks, I'm wondering if it makes sense to you removing this block for now and working that our in a follow up patch.

[1] https://github.com/openstack-k8s-operators/lib-common/blob/main/modules/openstack/volume.go#L35